src/checkers/inference/model/serialization/CnfVecIntSerializer.java

```diff
@@ -236,6 +237,11 @@ public boolean emptyClause(VecInt ... clauses) {
         return null;
     }
 
+    @Override
```

Comment:
move the whole method below so it follows the order set in Serializer.java, and same for the other concrete serializers.

Reply:
What's the benefit of this?

Reply:
If you want hash code to be consistent with the rest, then lets keep ordering consistent too

Reply:
later.